apps/application-system/api/src/app/modules/application/utils/pdf.ts

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
I hate this data uri string, but it what we are working with in this pdf file builder. Wondering if it would be nicer to move to another place to hide the uglyness 🙈  Or if I should keep it as close to the code as possible since it is not being reused.

Reply:
Well, this is not ideal but I don't have a preference if it should be here or in an external file 😅